nidx/src/import_export.rs

```diff
@@ -346,12 +340,6 @@ mod tests {
         let shard_dest = Shard::get(&meta_dest.pool, shard.id).await?;
         assert_eq!(shard_dest.kbid, shard.kbid);
         assert_eq!(Index::for_shard(&meta_dest.pool, shard.id).await?.len(), 3);
-        assert_eq!(
-            sqlx::query_scalar!("SELECT COUNT(*) AS \"cnt!\" FROM deletions")
-                .fetch_one(&meta_dest.pool)
```

Comment:
I want to keep this. This ensures deletions are exported+imported. We can just add some random deletion to the resource being indexed at the beginning of the test (maybe just set tests_to_delete to some random key).